extension/src/plots/vega/util.ts

```diff
@@ -255,16 +255,16 @@ const truncateTitle = (
 
 export const truncateVerticalTitle = (
   title: Text | Title,
-  width: number,
-  height: number
+  width = 2,
+  height = 2
```

Comment:
It was great having this on the extension side, but since we rely so much on screen size now. I think it'll be best to move this back on the webview side.